docs/select/README.md

```diff
@@ -3,13 +3,26 @@ Traditional retrieval of objects is always as whole entities, i.e GetObject for
 
 You can use the Select API to query objects with following features:
 
-- CSV, JSON and Parquet - Objects must be in CSV, JSON, or Parquet format.
+- Objects must be in CSV, JSON, or Parquet(*) format. 
 - UTF-8 is the only encoding type the Select API supports.
 - GZIP or BZIP2 - CSV and JSON files can be compressed using GZIP or BZIP2. The Select API supports columnar compression for Parquet using GZIP, Snappy, LZ4. Whole object compression is not supported for Parquet objects.
 - Server-side encryption - The Select API supports querying objects that are protected with server-side encryption.
 
 Type inference and automatic conversion of values is performed based on the context when the value is un-typed (such as when reading CSV data). If present, the CAST function overrides automatic conversion.
 
+The [mc sql](https://docs.min.io/docs/minio-client-complete-guide.html#sql) command can be used for executing queries using the command line. 
+
+(*) Parquet is disabled on the MinIO server by default. See below how to enable it.
+
+## Enabling Parquet Format
+
+Parquet is DISABLED by default since hostile crafted input can easily crash the server.
+
+If you are in a controlled environment where it is safe to assume no hostile content can be uploaded to your cluster you can safely enable Parquet.
+To enable Parquet set the environment variable `MINIO_SELECT_PARQUET=on`.
```

Comment:
can we name this as 
```suggestion
To enable Parquet set the environment variable `MINIO_API_SELECT_PARQUET=on`.
```

so that we can keep the ENV heirarchy properly with `MINIO_API`